packages/contract-addresses/src/index.ts

```diff
@@ -53,15 +56,17 @@ const networkToAddresses: { [networkId: number]: ContractAddresses } = {
         staking: NULL_ADDRESS,
         stakingProxy: NULL_ADDRESS,
         devUtils: NULL_ADDRESS,
+        erc20BridgeProxy: NULL_ADDRESS,
     },
     3: {
         erc20Proxy: '0xb1408f4c245a23c31b98d2c626777d4c0d766caa',
         erc721Proxy: '0xe654aac058bfbf9f83fcaee7793311dd82f6ddb4',
         zrxToken: '0xff67881f8d12f372d91baae9752eb3631ff0ed00',
         etherToken: '0xc778417e063141139fce010982780140aa0cd5ab',
         exchangeV2: '0xbff9493f92a3df4b0429b6d00743b3cfb4c85831',
-        exchange: '0x725bc2f8c85ed0289d3da79cde3125d33fc1d7e6',
-        assetProxyOwner: '0xdcf20f7b447d51f2b3e5499b7f6cbbf7295a5d26',
+        exchange: '0xc56388332ddfc98701fefed94535100c6166956c',
```

Comment:
Nit: I think that noting that this exchange is version 3 of the protocol is worth a comment. It might be a bit confusing to the uninitiated that this exchange is really *exchangeV3* rather than *exchangeV1*.

Reply:
I think the plan was to remove `exchangeV2` after mainnet migrations, but we still need it until then.